index.md

```diff
@@ -16,7 +16,8 @@
 - [[CTV15]](https://eprint.iacr.org/2015/377)
 - [ZKBoo [GMO16]](https://eprint.iacr.org/2016/163.pdf)
 - [[Groth16]](https://eprint.iacr.org/2016/260.pdf)
-- [Bootle proofs [BCC+16]](https://eprint.iacr.org/2016/263.pdf)
+- [BCC+16](https://eprint.iacr.org/2016/263.pdf)
```

Comment:
ACK on this change (I was just using what I'd heard other people refer to the original paper's proof system as).